src/main/java/org/folio/dew/batch/bulkedit/jobs/processidentifiers/BulkEditProcessIdentifiersJobConfig.java

```diff
@@ -0,0 +1,91 @@
+package org.folio.dew.batch.bulkedit.jobs.processidentifiers;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.des.domain.dto.ExportType;
+import org.folio.dew.batch.CsvWriter;
+import org.folio.dew.batch.JobCompletionNotificationListener;
+import org.folio.dew.domain.dto.UserFormat;
+import org.folio.dew.domain.dto.ItemIdentifier;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.core.launch.support.RunIdIncrementer;
+import org.springframework.batch.item.file.FlatFileItemReader;
+import org.springframework.batch.item.file.FlatFileItemWriter;
+import org.springframework.batch.item.file.LineMapper;
+import org.springframework.batch.item.file.builder.FlatFileItemReaderBuilder;
+import org.springframework.batch.item.file.mapping.BeanWrapperFieldSetMapper;
+import org.springframework.batch.item.file.mapping.DefaultLineMapper;
+import org.springframework.batch.item.file.transform.DelimitedLineTokenizer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.FileSystemResource;
+
+@Configuration
+@RequiredArgsConstructor
+public class BulkEditProcessIdentifiersJobConfig {
+  private static final int CHUNKS = 100;
+  private static final long ZERO = 0L;
+
+  private final JobBuilderFactory jobBuilderFactory;
+  private final StepBuilderFactory stepBuilderFactory;
+  private final BulkEditItemProcessor bulkEditItemProcessor;
+
+  @Bean
+  @StepScope
+  public FlatFileItemReader<ItemIdentifier> csvItemIdentifierReader(
+    @Value("#{jobParameters['fileName']}") String uploadedFileName) {
```

Comment:
identifiersFileName parameter was renamed to fileName cause this file can be a user records file too.